lib/init-exe/build.zig

```diff
@@ -63,5 +63,5 @@ pub fn build(b: *std.Build) void {
     // the `zig build --help` menu, providing a way for the user to request
     // running the unit tests.
     const test_step = b.step("test", "Run unit tests");
-    test_step.dependOn(&exe_tests.step);
+    test_step.dependOn(&exe_tests.run().step);
```

Comment:
I think `exe_tests.run()` is marked as deprecated. https://github.com/ziglang/zig/blob/37f6f7990e882ac9869513cf920f8cd0c4850039/lib/std/Build/CompileStep.zig#L551

I haven't tested the following, but I think it would be better to have something like
```zig
    // Creates a step for unit testing.
    const exe_tests = b.addTest(.{
        .root_source_file = .{ .path = "src/main.zig" },
        .target = target,
        .optimize = optimize,
    });
    const exe_tests_run = b.addRunArtifact(exe_tests);

    // Similar to creating the run step earlier, this exposes a `test` step to
    // the `zig build --help` menu, providing a way for the user to request
    // running the unit tests.
    const test_step = b.step("test", "Run unit tests");
    test_step.dependOn(&exe_tests_run.step);
```

Reply:
You are right and it works. Should I create a new pull request?

Reply:
no reason to create a new one. just force push to this branch.